### ISSUE TITLE
Problems with documentation about 'new sound' 

### ISSUE THREAD

**Issue by ToonTalk** [jmoenig/Snap#3042]
The only sentence about it is

![image](https://user-images.githubusercontent.com/2138696/166099829-9427487e-162f-4bd4-93a7-d212038b3219.png)

1. There is no "new picture" - should be 'new costume' 
2. This doesn't really explain the block very well (and 'new costume' is documented with only "The new costume
block creates a costume by combining a bitmap, a width, and a height.")
3. You can't search for 'new sound' in the PDF (other than finding it in the index)

Am I right that one motivation for these blocks is if the costume or sound is to be used repeatedly there is a performance cost (in time and memory) for using a bitmap or list of samples?

And as is increasingly common, there is no 'help' for these blocks.

**Comment by brianharvey** [comment]
Yeah, yeah, I'm way behind on documentation.  I'll try to catch up.
